www/docs/migration/migrate-from-v9-to-v10.mdx

```diff
@@ -588,3 +588,12 @@ const MyComponent = () => {
 ### Migrate custom error formatters
 
 You will need to move the contents of your `formatError()` into your root `t` router. See the [Error Formatting docs](error-formatting) for more.
+
+### Renaming of @trpc/react to @trpc/react-query
+
+The `@trpc/react` package has been renamed to `@trpc/react-query` to better reflect the fact that it's a thin wrapper around `react-query` and to better suit the upcoming RSC as that it won't be as needed as before. If you're using `@trpc/react` you'll need to remove it and install `@trpc/react-query` instead and also update your imports:
```

Comment:
As these are docs is it worth expanding what RSC is?

I'm guessing React Server Components are meant here? 
```suggestion
The `@trpc/react` package has been renamed to `@trpc/react-query` to better reflect the fact that it's a thin wrapper around `react-query` and also in anticipation of the upcoming React Server Components (RSC) release as this package won't be essential for react use going forwards. If you're using `@trpc/react` you'll need to remove it and install `@trpc/react-query` instead and also update your imports:
```

Reply:
I like this phrasing more, although maybe we shouldn't jump to conclusions and straight out say `it won't be as useful` and just open up for a potential react package that isn't bound to react-query?

Reply:
I do think it's a bit "hidden" away at the bottom? This is quite a "big breaking" change and should be further up imo

Reply:
How about :

```suggestion
The `@trpc/react` package has been renamed to `@trpc/react-query`. This is to reflect it is a thin wrapper around `react-query` as well as making room for instances where trpc may be used in react without the `@trpc/react-query` package; such as in use with upcoming React Server Components (RSC's) or in use with other data fetching library adapters. If you're using `@trpc/react` you'll need to remove it and install `@trpc/react-query` instead as well as updating your imports:
```